pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/SelectTupleElementTransformFunction.java

```diff
@@ -84,6 +87,23 @@ public String getName() {
     return _name;
   }
 
+  @Override
+  public RoaringBitmap getNullBitmap(ValueBlock valueBlock) {
+    RoaringBitmap bitmap = _arguments.get(0).getNullBitmap(valueBlock);
+    for (int i = 1; i < _arguments.size(); i++) {
+      RoaringBitmap curBitmap = _arguments.get(i).getNullBitmap(valueBlock);
+      if (bitmap != null && curBitmap != null) {
+        bitmap.and(curBitmap);
+      } else {
+        bitmap = null;
+      }
+    }
+    if (bitmap == null || bitmap.isEmpty()) {
```

Comment:
This check can be implemented before for loop as well

Reply:
yeah. good catch. updated the code.